### PR TITLE
write tests for delete note feature

### PR DESCRIPTION
**Pull request by apratt101300**
Created a couple of tests for the delete note feature.

@John151 @Nelwell @abdiawali @PickleEric

### CODE REVIEW

File: lmn/tests/test_views.py [hunk 1]
```diff
@@ -469,6 +469,24 @@ def test_correct_templates_uses_for_notes(self):
...
 
 
+class TestUserNotes(TestCase):
+
+    fixtures = [ 'testing_users', 'testing_artists', 'testing_venues', 'testing_shows', 'testing_notes' ]
+
+    def test_delete_own_note(self):
+        self.client.force_login(User.objects.first())
+        response = self.client.post(reverse('delete_note', args=(1,)), follow=True)
+        note_2 = Note.objects.filter(pk=1).first()
```
Comment (Nelwell) [inline lmn/tests/test_views.py:479]: Should change name of variable note_2 to just note if using pk=1? I think note_2 would represent a pk=2.

File: lmn/tests/test_views.py [hunk 2]
```diff
@@ -469,6 +469,24 @@ def test_correct_templates_uses_for_notes(self):
         self.assertTemplateUsed(response, 'lmn/notes/new_note.html')
 
 
+class TestUserNotes(TestCase):
+
+    fixtures = [ 'testing_users', 'testing_artists', 'testing_venues', 'testing_shows', 'testing_notes' ]
+
+    def test_delete_own_note(self):
+        self.client.force_login(User.objects.first())
+        response = self.client.post(reverse('delete_note', args=(1,)), follow=True)
```
Comment (Nelwell) [inline lmn/tests/test_views.py:478]: might not be significant, but response variable is unused in test_delete_own_note so can probably remove line 478 altogether here.

**Review by Nelwell — COMMENTED** [review]
Looks good, made a couple small suggestions.

File: lmn/tests/test_views.py (hunk 1 above)
Comment (apratt101300) [reply lmn/tests/test_views.py]: Good point, I just changed it. Thanks!

**Comment by apratt101300** [comment]
@Nelwell I made the suggested changes. Let me know if you see anything else, otherwise, you can feel free to merge. Thanks!

**Review by PickleEric — APPROVED** [review]
Nice work on the tests. Look like they are in working order

**Comment by Nelwell** [comment]
These tests look good. Tests related to this feature are passing and variable names are meaningful/representative of their purpose.

**Pull request merged**